### PR TITLE
Updated commands to `oc adm drain node`

### PR DESCRIPTION
**Pull request by ahardin-rh**
Addresses https://github.com/openshift/openshift-docs/issues/5613

@pravisankar Can you please check the syntax here?

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@pravisankar PTAL

File: admin_guide/manage_nodes.adoc [hunk 1]
```diff
@@ -245,22 +245,19 @@ To list pods that will be migrated without actually performing the evacuation,
 use the `--dry-run` option:
 
 ----
-$ oadm manage-node <node1> <node2> \
-    --evacuate --dry-run [--pod-selector=<pod_selector>]
+$ oadm drain <node1> <node2>  --dry-run [--pod-selector=<pod_selector>]
```
Comment (pravisankar) [inline admin_guide/manage_nodes.adoc]: --pod-selector is not a valid option for `oadm drain`.
Supported options:
```
 Usage:
  oadm drain NODE [options]

Examples:
  # Drain node "foo", even if there are pods not managed by a ReplicationController, ReplicaSet, Job, DaemonSet or StatefulSet on it.
  $ oadm drain foo --force
  
  # As above, but abort if there are pods not managed by a ReplicationController, ReplicaSet, Job, DaemonSet or StatefulSet, and use a grace period of 15 minutes.
  $ oadm drain foo --grace-period=900

Options:
      --delete-local-data=false: Continue even if there are pods using emptyDir (local data that will be deleted when the node is drained).
      --dry-run=false: If true, only print the object that would be sent, without sending it.
      --force=false: Continue even if there are pods not managed by a ReplicationController, ReplicaSet, Job, DaemonSet or StatefulSet.
      --grace-period=-1: Period of time in seconds given to each pod to terminate gracefully. If negative, the default value specified in the pod will be used.
      --ignore-daemonsets=false: Ignore DaemonSet-managed pods.
      --timeout=0s: The length of time to wait before giving up, zero means infinite
```

**Comment by ahardin-rh** [comment]
@pravisankar Thanks for your guidance. This is now updated. PTAL.

File: admin_guide/manage_nodes.adoc [hunk 2]
```diff
@@ -245,27 +245,45 @@ To list pods that will be migrated without actually performing the evacuation,
...
 ----
-$ oadm manage-node <node1> <node2> \
-    --evacuate --dry-run [--pod-selector=<pod_selector>]
+$ oadm drain <node1>  --dry-run=true
 ----
 
+If `dry-run` is set to `true`, you only print the object that would be sent,
+without sending it.
+
 To actually evacuate all or selected pods on one or more nodes:
```
Comment (pravisankar) [inline admin_guide/manage_nodes.adoc]: `To actually evacuate all pods on one or more nodes:` (currently there is no option to select pods)

**Review by pravisankar — COMMENTED** [review]
I'm not sure if this doc holds true for enterprise 3.5, 3.6, 3.7 and 3.9.
At least latest origin code (3.9) currently does not show `--dry-run` option but 3.9 is not released yet. Currently k8s rebase is in progress (https://github.com/openshift/origin/pull/17576) and we may have `--pod-selector` and `selector` options (https://github.com/openshift/origin/pull/17616) and also `--dry-run` option (https://github.com/openshift/origin/pull/16333).
I would recommend holding off publishing this change for 3.9 until we have final build ready and make sure all the `oadm drain` flags are same in 3.5, 3.6 and 3.7.

**Comment by ahardin-rh** [comment]
@pravisankar Okay, thanks. I will check back in later on this.

File: admin_guide/manage_nodes.adoc [hunk 3]
```diff
@@ -245,27 +245,45 @@ To list pods that will be migrated without actually performing the evacuation,
 use the `--dry-run` option:
 
 ----
-$ oadm manage-node <node1> <node2> \
-    --evacuate --dry-run [--pod-selector=<pod_selector>]
+$ oadm drain <node1>  --dry-run=true
```
Comment (miminar) [inline admin_guide/manage_nodes.adoc]: `oadm` has been removed:  openshift/origin#17396
it's being re-added already but with a fat decprecated warning though
`oc adm` is preferred

**Comment by miminar** [comment]
:+1:

I've just received complaints about this from a customer. I'm pleased to see it being fixed.

**Comment by tmorriso-rh** [comment]
@ahardin-rh Any update on this? I have a bug that is related: https://bugzilla.redhat.com/show_bug.cgi?id=1477380

Cc: @vikram-redhat

**Comment by ahardin-rh** [comment]
@tmorriso-rh Per discussion earlier in this PR, we decided to hold off on merging this until there is a final build ready so that we can confirm that `oc adm drain` flags are the same in 3.5, 3.6, and 3.7.
@pravisankar Any updates on this?

**Comment by pravisankar** [comment]
`drain` is an upstream k8s CLI command, corresponding kube docs already available but depending on when we do k8s rebase in openshift, available CLI options may vary. I'm guessing we are also documenting these commands to exactly show available command options for a particular release.

This is how `drain` command was evolved:
Options available in 3.5 and 3.6 release:
```
      --delete-local-data=false: Continue even if there are pods using emptyDir (local data that will be deleted when
the node is drained).
      --force=false: Continue even if there are pods not managed by a ReplicationController, ReplicaSet, Job, DaemonSet
or StatefulSet.
      --grace-period=-1: Period of time in seconds given to each pod to terminate gracefully. If negative, the default
value specified in the pod will be used.
      --ignore-daemonsets=false: Ignore DaemonSet-managed pods.
      --timeout=0s: The length of time to wait before giving up, zero means infinite
```

3.7 release:
```
In addition to options in 3.6
--dry-run=false: If true, only print the object that would be sent, without sending it.
```

3.8 release:
```
In addition to options in 3.7
--pod-selector='': Label selector to filter pods on the node
option --dry-run was removed, not sure why that was done
```

3.9 release:
```
In addition to options in 3.8
[option '--dry-run' was re-introduced]
--dry-run=false: If true, only print the object that would be sent, without sending it.
 -l, --selector='': Selector (label query) to filter on
```

**Review by pravisankar — CHANGES_REQUESTED** [review]
We have enough information now to doc this command. Provided needed details

**Comment by ahardin-rh** [comment]
@pravisankar Thanks for your guidance! PTAL. This PR addresses 3.5 and 3.6. I will open companion PRs for 3.7 and 3.9 to add in the additional details for each. We are not releasing docs for 3.8, but it's interesting seeing the evolution. Thanks again for all of your help in getting this moved along!

File: admin_guide/manage_nodes.adoc [hunk 4]
```diff
@@ -241,31 +241,49 @@ controller] can be evacuated; the replication controllers create new pods on
 other nodes and remove the existing pods from the specified node(s). Bare pods,
 meaning those not backed by a replication controller, are unaffected by default.
 
-To list pods that will be migrated without actually performing the evacuation,
-use the `--dry-run` option:
+To actually evacuate all or selected pods on one or more nodes:
```
Comment (pravisankar) [inline admin_guide/manage_nodes.adoc]: To actually evacuate one or more nodes:
[Selecting pods needs `--pod-selector` and this option is not available in 3.5/3.6]

**Review by pravisankar — CHANGES_REQUESTED** [review]
Minor nit otherwise LGTM

File: admin_guide/manage_nodes.adoc (hunk 4 above)
Comment (ahardin-rh) [reply admin_guide/manage_nodes.adoc]: @pravisankar Updated. Thank you!

**Comment by ahardin-rh** [comment]
Updated per feedback. I will merge this and build off of it for 3.7 and 3.9 docs in separate PRs. Revision histories for each will be applied in the cherry-pick PRs.

**Pull request merged**